plaso/cli/helpers/elastic_output.py

```diff
@@ -121,6 +127,24 @@ def ParseOptions(cls, options, output_module):
     elastic_user = cls._ParseStringOption(options, 'elastic_user')
     elastic_password = cls._ParseStringOption(options, 'elastic_password')
     use_ssl = getattr(options, 'use_ssl', False)
+    additional_fields_string = cls._ParseStringOption(
+        options, 'additional_fields')
+
+    if additional_fields_string:
+      try:
+        additional_fields = json.loads(additional_fields_string)
+      except (json.JSONDecodeError, ValueError) as exc:
```

Comment:
style nit: do not use abbreviations like `exc`

Reply:
done